File: Backend/ApiGateWay/Domain/Models/TestRequests/ChangeStatusTestRequest.cs

using Domain.Enums;

namespace Domain.Models.TestRequests
{
    public class ChangeStatusTestRequest
    {
        public TestRequestsStatus Status { get; set; }
        public string Message { get; set; }=string.Empty;
        public Attachment? Attachment{ get; set; }
        public int IdUser { get; set; }            
    }
}

File: FrontEnd/src/app/test-catalog/interface/change-status-test-request.ts
import { Attachment } from "../../shared/interfaces/test-interfaces/attachment.interface";
import { TestRequestEnum } from "../../test-requests/Enums/test-request.enum";

export interface ChangeStatusTestRequest {
    status: TestRequestEnum;
    message: string;
    attachment?: Attachment;
    idUser?: number;
    
 }
